src/Faker/Provider/pt_BR/Payment.php

```diff
@@ -4,6 +4,122 @@
 
 class Payment extends \Faker\Provider\Payment
 {
+    /**
+     * @var array list of Brazilian banks, source: http://www.febraban.org.br/bancos.asp
```

Comment:
the content is "© FEBRABAN - Todos os direitos reservados" and can't be redistributed in a MIT license project like Faker